component/componenttest/obsreporttest.go

```diff
@@ -38,6 +37,7 @@ const (
 )
 
 type TestTelemetry struct {
+	// Deprecated: [0.93.0] use TelemetrySettingsFunc() instead.
 	component.TelemetrySettings
```

Comment:
Given that users need to adapt to the move it anyway, should we hide this field in the new package and expose the function under this name instead of `TelemetrySettingsFunc`?

Reply:
Good point, I've updated the code